main/webapp/modules/core/scripts/project.js

```diff
@@ -39,7 +39,23 @@ var Refine = {};
 
 I18NUtil.init("core");
 
+/**
+ * Requests a CSRF token and calls the supplied callback.
+ * @public
+ * @param {function(string)} onCSRF - The callback to call with the token.
+ */
 Refine.wrapCSRF = CSRFUtil.wrapCSRF;
+
+/**
+ * Performs a POST request where an additional CSRF token is automatically attached.
+ * @public
+ * @param {string} url - The URL to send the request to.
+ * @param {data} data - The data to send with the request.
```

Comment:
Looking at the implementation of `CSRFUtil.wrapCSRF`, it looks like this could be narrowed down to `{(Object|string)}`.